web/src/user_group_components.ts

```diff
@@ -53,14 +58,15 @@ export function setup_permissions_dropdown(
         },
     });
     if (for_group_creation) {
-        settings_components.set_new_group_can_mention_group_widget(can_mention_group_widget);
+        if (setting_name === "can_mention_group") {
+            settings_components.set_new_group_can_mention_group_widget(group_setting_widget);
+        } else {
+            settings_components.set_new_group_can_manage_group_widget(group_setting_widget);
```

Comment:
Can update this to be a single function for all the settings as a follow-up.

Reply:
Sounds reasonable.